toxcore/Messenger.c

```diff
@@ -2169,7 +2202,14 @@ static int m_handle_packet(void *object, int i, const uint8_t *temp, uint16_t le
     uint32_t data_length = len - 1;
 
     if (m->friendlist[i].status != FRIEND_ONLINE) {
-        if (packet_id == PACKET_ID_ONLINE && len == 1) {
+        if (packet_id == PACKET_ID_ONLINE) {
+
```

Comment:
Remove empty line.

Reply:
The line is removed, read the line numbers, github is just weird.

Reply:
Ohhhhhhhh that line